cub/test/c2h/custom_type.cuh

```diff
@@ -56,6 +56,17 @@ public:
 
 };
 
+template <std::size_t TotalSize>
+struct huge_data_t
+{
+  template <class CustomType>
+  class data_t
```

Comment:
suggestion: use `type` for metafunction return and probably utilize `huge_data_t` name for alias `using huge_data_t = huge_data<TotalSize>::type`